css/templates/e-commerce/page.css

```diff
@@ -5,51 +5,265 @@
 @import 'page-vars';
 
 body {
-  font-size: var(--narrow-body-font-size);
-}
-
-@media (--breakpoint-md) {
-  body {
-    font-size: var(--wide-body-font-size);
-  }
+  font-size: var(--body-font-size);
+  line-height: initial;
 }
 
 /* TYPOGRAPHY */
 .h3 {
   color: var(--color-primary);
+}
+
+.h7 {
+  font-size: var(--h7);
+}
+
+h1, h2, h3, h4, h5, h6, h7,
+.h1, .h2, .h3, .h4, .h5, .h6, .h7 {
   line-height: normal;
-  padding-bottom: var(--space-1);
+  letter-spacing: 0;
+  font-family: var(--title-font);
+  text-transform: uppercase;
+  font-weight: bold;
+  color: var(--color-black);
+}
+
+.mb5 {
+  margin-bottom: var(--space-6);
+}
+
+.pb5 {
+  padding-bottom: var(--space-6);
+}
+
+@media (--breakpoint-md) {
+  .md-h1 {
+    font-size: var(--h1);
+  }
+
+  .md-h2 {
+    font-size: var(--h2);
+  }
+
+  .md-h3 {
+    font-size: var(--h3);
+  }
+
+  .md-h4 {
+    font-size: var(--h4);
+  }
+
+  .md-h5 {
+    font-size: var(--h5);
+  }
+
+  .md-h6 {
+    font-size: var(--h6);
+  }
+
+  .md-h7 {
+    font-size: var(--h7);
+  }
+
+  .md-px4 {
```

Comment:
I've added custom padding classes akin to Bass CSS when needed. This ensured the CSS was kept small since small immutable classes are created rather than custom component names classes.